extensions/ql-vscode/src/model-editor/model-editor-module.ts

```diff
@@ -204,7 +203,6 @@ export class ModelEditorModule extends DisposableObject {
             queryDir,
             db,
             modelFile,
-            Mode.Application,
```

Comment:
Another one bites the dust 😆 

I love seeing the constructor getting smaller as we rely more on the modeling store.